src/hooks/useI18n.ts

```diff
@@ -11,17 +15,41 @@ const useI18n = () => {
    * TODO: Figure out if the context is used and if not, remove it.
    */
   return useCallback(
-    (translationId: number, fallback: string) => {
-      if (translations[0] === 'error') {
+    (translationId: number, fallback: string, data: ContextData = {}) => {
+      if (translations.length === 0) {
         return fallback
       }
-      if (translations.length > 0) {
-        return getTranslation(translations, translationId, fallback)
+
+      const foundTranslation = translations.find((translation) => {
```

Comment:
@hachiojidev  can you have a look at this change ?

Reply:
@hachiojidev kindly remainder ☺️